app/src/main/java/org/p2p/wallet/moonpay/serversideapi/MoonpayServerSideApi.kt

```diff
@@ -20,5 +21,5 @@ interface MoonpayServerSideApi {
     @DELETE("api/v3/sell_transactions/{transactionId}")
     suspend fun cancelSellTransaction(
         @Path("transactionId") sellTransactionId: String
-    )
+    ): Response<Unit>
```

Comment:
Response<Unit> ? strange move

Reply:
i know right?
retrofit stuff
204 returns empty body and he fails to parse it without this